sceptre/diffing/diff_writer.py

```diff
@@ -203,7 +211,18 @@ def has_config_difference(self) -> bool:
     def has_template_difference(self) -> bool:
         return len(self.template_diff) > 0
 
+    def _color_diff(self, diff):
+        for line in diff:
+            if line.startswith('+'):
+                yield Fore.GREEN + line + Fore.RESET
+            elif line.startswith('-'):
+                yield Fore.RED + line + Fore.RESET
+            elif line.startswith('^'):
+                yield Fore.BLUE + line + Fore.RESET
+            else:
+                yield line
```

Comment:
If you do this, you'll need to regard the `--no-colour` configuration flag that Sceptre accepts and is always added to the context object.